oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/CommitBuilderTest.java

```diff
@@ -119,8 +119,8 @@ public void addNodePathNull() {
         CommitBuilder builder = new CommitBuilder(ns, null);
         try {
             builder.addNode((Path) null);
-            expectNPE();
-        } catch (NullPointerException e) {
+            expectIllegalArgumentException();
+        } catch (IllegalArgumentException e) {
```

Comment:
same

Reply:
The tests fail because an IllegalArgumentException is actually thrown, so I have modified them to pass.